### PR TITLE
Debugging Checkpoints

### PR DESCRIPTION
**Pull request by l-monninger**
### Description
Handles reward methods by setting the rate to 0.

### Test Plan
1. Tested e2e with movementlabsxyz/movement.

### CODE REVIEW

File: aptos-move/vm-genesis/src/lib.rs [hunk 1]
```diff
@@ -417,12 +417,12 @@ fn initialize(
     // Calculate the per-epoch rewards rate, represented as 2 separate ints (numerator and
     // denominator).
     let rewards_rate_denominator = 1_000_000_000;
-    // Fold division of the percentage by 100 into multiplication with the denominator
-    // to minimize rounding errors due to integer division.
-    let rewards_rate_numerator = genesis_config.rewards_apy_percentage
-        * (rewards_rate_denominator / 100)
-        * genesis_config.epoch_duration_secs
-        / NUM_SECONDS_PER_YEAR;
+    let _num_epochs_in_a_year = NUM_SECONDS_PER_YEAR / genesis_config.epoch_duration_secs;
```
Comment (mzabaluev) [inline aptos-move/vm-genesis/src/lib.rs:420]: This will still trigger division by zero if we set `epoch_duration_secs` to a value longer than a year, so perhaps comment this out completely.
The code I replaced it with computes the numerator correctly for a wider range of values.

File: aptos-move/vm-genesis/src/lib.rs [hunk 2]
```diff
@@ -417,12 +417,12 @@ fn initialize(
...
+    // Multiplication before division to minimize rounding errors due to integer division.
+    /*let rewards_rate_numerator = (genesis_config.rewards_apy_percentage * rewards_rate_denominator
+        / 100)
+        / num_epochs_in_a_year;*/
+    let rewards_rate_numerator = 0; // todo: feature flag this for Movement Labs
```
Comment (mzabaluev) [inline aptos-move/vm-genesis/src/lib.rs:425]: A less intrusive way to effect this is to set `rewards_apy_percentage` to 0 in the config and relax the check in the `validate_genesis_config` function. That may be cleaner, although I don't see the config parameter used as source of truth for anything else in aptos-core code.

**Review by mzabaluev — APPROVED** [review]
Corrected myself in the comments and I think it can go ahead despite these concerns.

File: aptos-move/vm-genesis/src/lib.rs (hunk 2 above)
Comment (l-monninger) [reply aptos-move/vm-genesis/src/lib.rs:425]: Yeah, I'm sure there's a way. We can put in a small ticket for it. This is just to keep things moving and make sure we're not too all over the place.
